templates/auth0-login-form.php

```diff
@@ -25,12 +25,13 @@
 $custom_signup_fields = (string) trim( $auth0_options->get( 'custom_signup_fields' ) );
 
 if ( $custom_js || $custom_signup_fields ) {
```

Comment:
This line can be deleted. Both cases are accepted below.

Reply:
I don't want the empty script tags though